api/migrations/add_slug_to_not_null.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
Change the file name. Add timestamp. Migrations should be generated using sequelize cli.